docs/handbook/getting-started.md

```diff
@@ -1933,12 +1949,17 @@ The beadm command impacts the non-global zones in your boot environments as foll
 
 ## The X-Window system
 
-Nearly all cards can use the VESA driver, and are therefore supported for 2D.
+### Video card support (2D)
+
+* Nearly all cards can use the VESA driver, and are therefore supported for 2D.
+* 2D acceleration for NVidia GPU's is provided by the Nvidia binary driver.
+* Work has begun to provide Radeon (2D) acceleration support for both legacy and modern AMD GPU's.
```

Comment:
Yes. The issue is in kernel driver, which never worked. Honestly, it worked for some ancient ATI video adapters, but it is irrelevant now. I think Ken is working on supporting current 2D driver on newer Xorg. As we were recently pointed out, we just have to test latest pkgsrc patches...